src/System.Reflection/tests/MethodInfoTests.cs

```diff
@@ -384,15 +384,15 @@ public void Attributes()
         {
             MethodInfo methodInfo = GetMethod(typeof(MI_SubClass), "ReturnVoidMethod");
             MethodAttributes attributes = methodInfo.Attributes;
-            Assert.NotNull(attributes);
+            Assert.True(attributes.HasFlag(MethodAttributes.Public));
         }
 
         [Fact]
         public void CallingConvention()
         {
             MethodInfo methodInfo = GetMethod(typeof(MI_SubClass), "ReturnVoidMethod");
             CallingConventions callingConvention = methodInfo.CallingConvention;
-            Assert.NotNull(callingConvention);
+            Assert.True(callingConvention.HasFlag(CallingConventions.VarArgs));
```

Comment:
This seems to be wrong.

Reply:
As I was mentioning in comments, I’m having strangeness on my machine. I’m seeing tests pass that should not pass that fail during CI. I am going to resume this morning on my work computer.

Reply:
I was a bonehead, updated this to HasThis given its a local instance method in latest commit.